modules/serf/main.tf

```diff
@@ -0,0 +1,74 @@
+variable "num_nodes" { }
+variable "region" { }
+
+variable "image" {
+	default = "ubuntu-x86_64"
```

Comment:
You are correct. I forgot to fix during my merge. I'll update it with correct slug.

On January 31, 2016 12:12:44 PM EST, Carlos Amedee <notifications@github.com> wrote:
>> @@ -0,0 +1,74 @@
>> +variable "num_nodes" { }
>> +variable "region" { }
>> +
>> +variable "image" {
>> +	default = "ubuntu-x86_64"
>
>Doesn't seem to be a recognized image. 
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/criprooff/infrastructure/pull/3/files#r51363759

-- 
/thor